File: yangxj96-starter/yangxj96-starter-remote/src/main/java/io/github/yangxj96/starter/remote/clients/SystemFeignClient.java
package io.github.yangxj96.starter.remote.clients;

import io.github.yangxj96.starter.remote.fallback.SystemFeignClientFallback;
import org.springframework.cloud.openfeign.FeignClient;
import org.springframework.web.bind.annotation.GetMapping;

@FeignClient(name = "yangxj96-serve-system", fallback = SystemFeignClientFallback.class)
public interface SystemFeignClient {

    @GetMapping("/")
    String get();


    @GetMapping("/")
    String get2();
}

File: yangxj96-starter/yangxj96-starter-remote/src/main/java/io/github/yangxj96/starter/remote/clients/SystemFeignClient.kt
package io.github.yangxj96.starter.remote.clients

import io.github.yangxj96.starter.remote.fallback.SystemFeignClientFallback
import org.springframework.cloud.openfeign.FeignClient
import org.springframework.web.bind.annotation.GetMapping

@FeignClient(name = "yangxj96-serve-platform", fallback = SystemFeignClientFallback::class)
interface SystemFeignClient {

    @GetMapping("/demo/d1")
    fun get(): String

    @GetMapping("/demo/d2")
    fun demo2(): String

}
